### ISSUE TITLE
[Bug] SBA no longer hiding Skyblock Default Mana

### ISSUE THREAD

**Issue by Cryptic-Soldier** [Fix3dll/SkyblockAddons#50]
### Bug Description

According to everything I can find, SBA USED to disable/hide Skyblocks cyan text above the hot bar showing mana, mana use, when you change locations etc. Apparently it doesn't anymore.

See: https://i.imgur.com/M8oInNM.png in the red box.

I've checked all settings, and no matter what I can't get this text to go away.

### SBA Version

1.8.9

### Other Installed Mods

```md
# System Stats
[OS][Windows 11]
[CPU][16x AMD Ryzen 7 5800X 8-Core Processor ]
[Display][2560x1369 (NVIDIA Corporation)]
[GPU][NVIDIA GeForce RTX 3060/PCIe/SSE2]
[GPU Driver][4.6.0 NVIDIA 552.22]
[Maximum Memory][65457MB]
[Shaders][TRUE]
# Java Stats
[Java][1.8.0_412 64bit]
[Memory][ 31% 3920/12288MB]
[Memory Allocated][ 65% 8030MB]
# Game Stats
[FPS][197]
[Loaded Mods][20]
[Forge][11.15.1.2318]
[Optifine][TRUE]
# Neu Settings
[On SkyBlock][TRUE]
[Mod Version][2.3.2+e222ef31]
[Version Id][20302]
[SB Profile][Pineapple]
# Repo Stats
[Last Commit][dcbfe71b4bde584b12555d4c7b33e22904b1ddea]
[Repo Location][https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/tree/master]
[Using Backup][false]
[Loaded Items][6571]
[Lowest Bin API Location][moulberry.codes]
# Repo Files
[.gitattributes]
[.github][1]
[.gitignore]
[.vscode][2]
[constants][32]
[enforced_values][2]
[items][6571]
[LICENSE]
[mobs][220]
[neu-items-master.zip]
[README.md]
[update.json]
[update.json.asc]
# Mods Loaded
[Minecraft Coder Pack][minecraft.jar (9.19)]
[Forge Mod Loader][forge-1.8.9-11.15.1.2318-1.8.9-universal.jar (8.0.99.99)]
[Minecraft Forge][forge-1.8.9-11.15.1.2318-1.8.9-universal.jar (11.15.1.2318)]
[Essential][Essential (forge_1.8.9).processed.jar (1.3.2.8)]
[OneConfig][OneConfig (1.8.9-forge).jar (0.2.2-alpha212)]
[CrashPatch][CrashPatch-1.8.9-forge-2.0.1.jar (2.0.1)]
[ChatTriggers][ctjs-2.2.1-1.8.9.jar (2.2.1-1.8.9)]
[Dulkir Mod][DulkirMod-1.2.8.jar (1.2.8)]
[Dungeon Rooms Mod][Dungeon_Rooms-3.4.1.jar (3.4.1)]
[LunatriusCore][LunatriusCore-1.8.9-1.1.2.32-universal.jar (1.1.2.32)]
[NotEnoughUpdates][NotEnoughUpdates-2.3.2.jar (2.3.2+e222ef31)]
[Patcher][Patcher-1.8.9 (1.8.9).jar (1.8.9)]
[Schematica][Schematica-1.8.9-1.7.7.143-universal.jar (1.7.7.143)]
[Scrollable Tooltips][Scrollable Tooltips-1.4 (1.8.9).jar (LOCAL)]
[SkyblockAddons Unofficial][SkyblockAddons-1.9.0-beta.12.1+320-for-MC-1.8.9.jar (1.9.0-beta.12.1+320)]
[SkyblockExtras][SkyblockExtras-2.2.0-pre24.jar (2.2.0-pre24)]
[SkyHanni][SkyHanni-0.25.jar (0.25)]
[Skytils][Skytils-1.9.8.jar (1.9.8)]
[SoopyV2Forge][soopyv2forge-1.1.jar (1.1)]
[VolcAddons][VolcAddons-1.1.jar (1.1)]
```

### Operating System

Windows 11

**Comment by Fix3dll** [comment]
https://discord.com/channels/450878205294018560/472840295097303040/1261511523445248000Did you try “turn off revert health and defense” on SBE? If your answer is yes i will check it when i have free time.On 22 Jul 2024, at 09:36, Cryptic-Soldier ***@***.***> wrote:﻿
Bug Description
According to everything I can find, SBA USED to disable/hide Skyblocks cyan text above the hot bar showing mana, mana use, when you change locations etc. Apparently it doesn't anymore.
See: https://i.imgur.com/M8oInNM.png in the red box.
I've checked all settings, and no matter what I can't get this text to go away.
SBA Version
1.8.9
Other Installed Mods
# System Stats
[OS][Windows 11]
[CPU][16x AMD Ryzen 7 5800X 8-Core Processor ]
[Display][2560x1369 (NVIDIA Corporation)]
[GPU][NVIDIA GeForce RTX 3060/PCIe/SSE2]
[GPU Driver][4.6.0 NVIDIA 552.22]
[Maximum Memory][65457MB]
[Shaders][TRUE]
# Java Stats
[Java][1.8.0_412 64bit]
[Memory][ 31% 3920/12288MB]
[Memory Allocated][ 65% 8030MB]
# Game Stats
[FPS][197]
[Loaded Mods][20]
[Forge][11.15.1.2318]
[Optifine][TRUE]
# Neu Settings
[On SkyBlock][TRUE]
[Mod Version][2.3.2+e222ef31]
[Version Id][20302]
[SB Profile][Pineapple]
# Repo Stats
[Last Commit][dcbfe71b4bde584b12555d4c7b33e22904b1ddea]
[Repo Location][https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/tree/master]
[Using Backup][false]
[Loaded Items][6571]
[Lowest Bin API Location][moulberry.codes]
# Repo Files
[.gitattributes]
[.github][1]
[.gitignore]
[.vscode][2]
[constants][32]
[enforced_values][2]
[items][6571]
[LICENSE]
[mobs][220]
[neu-items-master.zip]
[README.md]
[update.json]
[update.json.asc]
# Mods Loaded
[Minecraft Coder Pack][minecraft.jar (9.19)]
[Forge Mod Loader][forge-1.8.9-11.15.1.2318-1.8.9-universal.jar (8.0.99.99)]
[Minecraft Forge][forge-1.8.9-11.15.1.2318-1.8.9-universal.jar (11.15.1.2318)]
[Essential][Essential (forge_1.8.9).processed.jar (1.3.2.8)]
[OneConfig][OneConfig (1.8.9-forge).jar (0.2.2-alpha212)]
[CrashPatch][CrashPatch-1.8.9-forge-2.0.1.jar (2.0.1)]
[ChatTriggers][ctjs-2.2.1-1.8.9.jar (2.2.1-1.8.9)]
[Dulkir Mod][DulkirMod-1.2.8.jar (1.2.8)]
[Dungeon Rooms Mod][Dungeon_Rooms-3.4.1.jar (3.4.1)]
[LunatriusCore][LunatriusCore-1.8.9-1.1.2.32-universal.jar (1.1.2.32)]
[NotEnoughUpdates][NotEnoughUpdates-2.3.2.jar (2.3.2+e222ef31)]
[Patcher][Patcher-1.8.9 (1.8.9).jar (1.8.9)]
[Schematica][Schematica-1.8.9-1.7.7.143-universal.jar (1.7.7.143)]
[Scrollable Tooltips][Scrollable Tooltips-1.4 (1.8.9).jar (LOCAL)]
[SkyblockAddons Unofficial][SkyblockAddons-1.9.0-beta.12.1+320-for-MC-1.8.9.jar (1.9.0-beta.12.1+320)]
[SkyblockExtras][SkyblockExtras-2.2.0-pre24.jar (2.2.0-pre24)]
[SkyHanni][SkyHanni-0.25.jar (0.25)]
[Skytils][Skytils-1.9.8.jar (1.9.8)]
[SoopyV2Forge][soopyv2forge-1.1.jar (1.1)]
[VolcAddons][VolcAddons-1.1.jar (1.1)]
Operating System
Windows 11
Crash Report
No response

**Comment by Cryptic-Soldier** [comment]
That setting was on. Toggling it off removed it. So that was my problem. This ended up being user error. Thank you very much for your help, and for continuing to work on this project. Hopefully we'll see this implemented into 1.21 or you can with with the Skyblocker devs to bring this stuff to that version.
